### PR TITLE
Feature/refactor simple test

### PR DESCRIPTION
**Pull request by tmeliotpg**

### CODE REVIEW

File: examples/readme.md [hunk 1]
```diff
@@ -11,15 +11,15 @@ git clone https://github.com/pagantis/orders-api-client.git .
 composer install
 ```
 
-Then you can run the web application using PHP's built-in server:
+Then you can run the web application using PHP's built-in server via Composer Scripts:
 
 ```
-php -S 0.0.0.0:8000 -t examples/
+composer run-script examples-server
```
Comment (pgarcess) [inline examples/readme.md]: After execute this script I receive this warning:
You made a reference to a non-existent script @putenv COMPOSER_PROCESS_TIMEOUT=0
You made a reference to a non-existent script @putenv PHP_CLI_SERVER_WORKERS=2

File: examples/readme.md [hunk 2]
```diff
@@ -11,15 +11,15 @@ git clone https://github.com/pagantis/orders-api-client.git .
...
 ```
 
-The web application is running at [http://localhost:8000/simpleTest.php](http://localhost:8000/simpleTest.php).
+The web application is running at [http://localhost:8000/index.php](http://localhost:8000/index.php).
```
Comment (pgarcess) [inline examples/readme.md]: Entering in this site I receive this message on screen:
Warning: require_once(../examples/utils/Helpers.php): failed to open stream: No such file or directory in /Users/pgarces/Project/libraries/ordersApiClient/examples/index.php on line 7

Fatal error: require_once(): Failed opening required '../examples/utils/Helpers.php' (include_path='.:/usr/local/Cellar/php@7.2/7.2.23/share/php@7.2/pear') in /Users/pgarces/Project/libraries/ordersApiClient/examples/index.php on line 7

File: examples/readme.md [hunk 3]
```diff
@@ -11,15 +11,15 @@ git clone https://github.com/pagantis/orders-api-client.git .
...
 
-To trace the execution in real time, you can open the log file using the command: 
+To trace the execution in real time, you can open the log file using the following Composer script: 
 ```
-tail -f examples/pagantis.log
+composer run-script watch-logs
```
Comment (pgarcess) [inline examples/readme.md]: After executing this script I receive this message on screen:
 [InvalidArgumentException]                          
  Script "watch-logs" is not defined in this package

File: composer.json [hunk 4]
```diff
@@ -13,6 +13,23 @@
         "phpunit/phpunit": "^4.8",
         "fzaninotto/faker": "^1.8"
     },
+
```
Comment (pgarcess) [inline composer.json]: We need to be compatible from php v5.3.
We have Circle CI procedure to validate the installation, this procedure answers with this message:
nstalling dependencies (including require-dev) from lock file
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for doctrine/instantiator 1.3.0 -> satisfiable by doctrine/instantiator[1.3.0].
    - doctrine/instantiator 1.3.0 requires php ^7.1 -> your PHP version (5.3.29) does not satisfy that requirement.
  Problem 2
    - Installation request for phpdocumentor/reflection-common 2.0.0 -> satisfiable by phpdocumentor/reflection-common[2.0.0].
    - phpdocumentor/reflection-common 2.0.0 requires php >=7.1 -> your PHP version (5.3.29) does not satisfy that requirement.
  Problem 3
    - Installation request for phpdocumentor/reflection-docblock 5.1.0 -> satisfiable by phpdocumentor/reflection-docblock[5.1.0].
    - phpdocumentor/reflection-docblock 5.1.0 requires ext-filter ^7.1 -> the requested PHP extension filter has the wrong version (0.11.0) installed.
  Problem 4
    - Installation request for phpdocumentor/type-resolver 1.1.0 -> satisfiable by phpdocumentor/type-resolver[1.1.0].
    - phpdocumentor/type-resolver 1.1.0 requires php ^7.2 -> your PHP version (5.3.29) does not satisfy that requirement.
  Problem 5
    - Installation request for symfony/yaml v3.4.38 -> satisfiable by symfony/yaml[v3.4.38].
    - symfony/yaml v3.4.38 requires php ^5.5.9|>=7.0.8 -> your PHP version (5.3.29) does not satisfy that requirement.
  Problem 6
    - doctrine/instantiator 1.3.0 requires php ^7.1 -> your PHP version (5.3.29) does not satisfy that requirement.
    - phpunit/phpunit-mock-objects 2.3.8 requires doctrine/instantiator ^1.0.2 -> satisfiable by doctrine/instantiator[1.3.0].
    - Installation request for phpunit/phpunit-mock-objects 2.3.8 -> satisfiable by phpunit/phpunit-mock-objects[2.3.8].

File: .gitignore [hunk 5]
```diff
@@ -1,2 +1,2 @@
 ### Composer template
-/vendor/
+/vendor/
```
Comment (pgarcess) [inline .gitignore]: Good practice is enter a line break at the end of a file, as the original one

File: src/Model/Order/Configuration.php [hunk 6]
```diff
@@ -30,7 +30,7 @@ class Configuration extends AbstractModel
     /**
      * @var array $allowedCountries
      */
-    private $allowedCountries = array('IT','ES','PT','FR');
```
[the remark this answers is not in the document]
Comment (tmeliotpg) [reply src/Model/Order/Configuration.php]: git checkout master src/Model/Order/Configuration.php and did not track changes from branch

File: src/Exception/HttpException.php [hunk 7]
```diff
@@ -66,9 +66,7 @@ public function __construct($statusCode = 500, $statusPhrase = null, array $head
             $statusPhrase = $this->status[$statusCode];
         }
         parent::__construct($statusPhrase, $statusCode);
-
```
[the remark this answers is not in the document]
Comment (tmeliotpg) [reply src/Exception/HttpException.php]: git checkout master src/Exception/HttpException.php  and did not track changes from branch

File: examples/readme.md (hunk 1 above)
Comment (tmeliotpg) [reply examples/readme.md]: removed fluff

File: examples/readme.md (hunk 3 above)
Comment (tmeliotpg) [reply examples/readme.md]: fixed typo

File: composer.json (hunk 4 above)
Comment (tmeliotpg) [reply composer.json]: removed useless library

File: .gitignore (hunk 5 above)
Comment (tmeliotpg) [reply .gitignore]: added line break

File: examples/readme.md (hunk 2 above)
Comment (tmeliotpg) [reply examples/readme.md]: fixed path

**Pull request closed without merging**